cmd/oci-unpack/main.go

```diff
@@ -117,6 +118,9 @@ func (v *unpackCmd) Run(cmd *cobra.Command, args []string) {
 
 	case image.TypeImage:
 		err = image.Unpack(args[0], args[1], v.ref)
+
+	default:
+		err = errors.New("Error media type")
```

Comment:
Here seems better to refer to https://github.com/opencontainers/image-tools/blob/master/cmd/oci-image-validate/main.go#L176, as fmt.Errorf("type %q unimplemented", v.typ) , and exit by `os.Exit(1)`.